bin/anthology/formatter.py

```diff
@@ -21,6 +21,7 @@
 from citeproc.source.json import CiteProcJSON
 import citeproc_styles
 import codecs
+import os
```

Comment:
Just a note, it might be nice to start slowly switching to `pathlib` instead of `os`.

Reply:
Agreed